packages/ralph/skills/golem-powers/figma-loop/SKILL.md

```
@@ -0,0 +1,153 @@
+
+---
+
+## RTL Quick Reference for Figma
+
+| Visual Position (RTL) | DOM Order | Tailwind |
+|----------------------|-----------|----------|
+| RIGHT | First | `items-start`, `justify-start` |
+| LEFT | Last | `items-end`, `justify-end` |
+
+**Button icons in RTL:**
+```tsx
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)